aces/Loris
# [Issue Tracker] Add upgrade script for module column

## Brief summary of changes

Issue Tracker has a `module` table which references IDs in the deleted LorisMenu table.

This PR adds a script to map the old LorisMenu IDs in this column to the corresponding IDs in the new `modules` table.

#### Testing instructions (if applicable)

1. Your database must have both the `LorisMenu` table and the `modules` table.
- If you've already installed `modules` and dropped `LorisMenu`, download and run [this SQL file](https://raw.githubusercontent.com/aces/Loris/2f07cc6d75a7f78201748ec917e963062972fa8a/SQL/0000-00-02-Menus.sql)
- If you still have `LorisMenu` and do not have `modules`, download and run [this SQL file](https://raw.githubusercontent.com/driusan/Loris/ec0981150a29bed86eece3b3a3452d9f166cf72a/SQL/0000-00-02-Modules.sql).

2. Run my script. It should display a before and after comparison of the values as well as the SQL statements you need to run, e.g. on Raisinbread:

```SQL
Issue ID: 16. Module before (`LorisMenu`): `dataquery/`. Module after (`modules`): `dataquery`
Issue ID: 19. Module before (`LorisMenu`): `examiner/`. Module after (`modules`): `examiner`
Issue ID: 20. Module before (`LorisMenu`): `media/`. Module after (`modules`): `media`
Issue ID: 26. Module before (`LorisMenu`): `examiner/`. Module after (`modules`): `examiner`
Issue ID: 27. Module before (`LorisMenu`): `conflict_resolver/`. Module after (`modules`): `conflict_resolver`
UPDATE `issues` SET module=14 WHERE issueID=16;
UPDATE `issues` SET module=17 WHERE issueID=19;
UPDATE `issues` SET module=29 WHERE issueID=20;
UPDATE `issues` SET module=17 WHERE issueID=26;
UPDATE `issues` SET module=9 WHERE issueID=27;
ALTER TABLE `issues` ADD FOREIGN KEY (moduleID) REFERENCES (modules(ID));
```


#### Link(s) to related issue(s)

* Resolves #5959
